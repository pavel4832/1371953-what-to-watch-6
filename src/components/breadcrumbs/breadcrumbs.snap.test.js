import React from 'react';
import {Provider} from 'react-redux';
import {render} from '@testing-library/react';
import {Router} from 'react-router-dom';
import {createMemoryHistory} from 'history';
import Breadcrumbs from './breadcrumbs';
import configureStore from 'redux-mock-store';

const mockStore = configureStore({});

it(`Should Breadcrumbs render correctly`, () => {
  const isReview = true;
  const history = createMemoryHistory();
  const {container} = render(
      <Provider store={mockStore({})}>
        <Router history={history}>
          <Breadcrumbs isReview={isReview} />
        </Router>
      </Provider>
  );
  expect(container).toMatchSnapshot();
});
