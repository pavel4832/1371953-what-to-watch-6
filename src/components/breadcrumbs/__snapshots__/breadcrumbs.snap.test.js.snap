// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Breadcrumbs render correctly 1`] = `
<div>
  <nav
    class="breadcrumbs"
  >
    <ul
      class="breadcrumbs__list"
    >
      <li
        class="breadcrumbs__item"
      >
        <a
          class="breadcrumbs__link"
          href="movie-page.html"
        >
          The Grand Budapest Hotel
        </a>
      </li>
      <li
        class="breadcrumbs__item"
      >
        <a
          class="breadcrumbs__link"
          href="/films/:id/review"
        >
          Add review
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
